apex/contrib/test/layer_norm/test_fast_layer_norm.py

```diff
@@ -232,29 +239,38 @@ def test_all_configs(self):
             49152,
             65536,
         ]
-
-        for h in hidden_sizes:
-            with self.subTest(f"hidden_size={h}"):
-                self.assertAll(_test_impl(256, 2, h, fp32, fp32))
-                self.assertAll(_test_impl(256, 2, h, fp16, fp16))
-                self.assertAll(_test_impl(256, 2, h, fp32, fp16))
-                self.assertAll(_test_impl(256, 2, h, bf16, bf16))
-                self.assertAll(_test_impl(256, 2, h, fp32, bf16))
+        use_1p = [
+            False,
+            True
+        ]
+        for is_1p in use_1p:
+            for h in hidden_sizes:
+                with self.subTest(f"hidden_size={h}, use_1p={is_1p}"):
+                    self.assertAll(_test_impl(256, 2, h, fp32, fp32, is_1p))
```

Comment:
Is there an existing test (already in Apex) that checks that custom LayerNorm (like FastLayerNormFN) behaves exactly the same as e.g. PyT LayerNorm? It would be nice to double check that our modification with adding +1 is mathematically correct. It seems that `_test_impl` uses `backward_` to check correctness, which is using the same +1 logic as the kernels.